app/javascript/hooks/forms/authentication/useSignUpForm.js

```diff
@@ -2,11 +2,37 @@ import * as yup from 'yup';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { useTranslation } from 'react-i18next';
 import { useForm } from 'react-hook-form';
+import { useMemo } from 'react';
+
+export function useSignUpFormValidation() {
+  return useMemo(() => (yup.object({
```

Comment:
This was made as a hook so that we can easily do localization from the validations level (if we decide to),